File: patterns/rule-deaktivierung/rules-python/test-dummy-mbr-humidifier.py
# Reine Test-Hilfsdatei, um "rule_deaktivierung.py" testen zu koennen -
# im echten Einsatz waere das eure tatsaechliche Luftbefeuchter-Regel.
from openhab import rule
from openhab.triggers import ItemCommandTrigger


@rule(uid="mbr_humidifier", name="MBR Humidifier (Test-Dummy)",
      triggers=[ItemCommandTrigger("DummyRuleTrigger")])
class MbrHumidifierDummy:
    def execute(self, module, input):
        pass

File: patterns/rule-deaktivierung/rules-js/_test_dummy_mbr_humidifier.js
// Reine Test-Hilfsdatei, um "rule-deaktivierung.js" testen zu koennen -
// im echten Einsatz waere das eure tatsaechliche Luftbefeuchter-Regel.
rules.JSRule({
  id: "mbr_humidifier",
  name: "MBR Humidifier (Test-Dummy)",
  triggers: [triggers.ItemCommandTrigger("DummyRuleTrigger")],
  execute: (event) => {
    // absichtlich leer
  }
});
